flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -71,15 +75,20 @@ def write_dataset(input: Channel,
                   name: Text = None
                   ) -> None:
     """
-    Write dataset to dataset operator. It can write dataset to external system.
-
-    :param input: Channel from the specific operator which generates data.
-    :param dataset_info: Information about the dataset which will be read. Its type can be DataSetMeta
-                         of py:class:`ai_flow.meta.dataset_meta.DatasetMeta` or Text or int. The dataset_info
-                         means name in the metadata service when its type is Text and it means id when its type is int.
-                         The ai flow will get the dataset from metadata service by name or id.
-    :param write_dataset_processor: The python user defined function in write dataset operator. User can write their own logic here.
-    :param name: Name of the read_dataset operator.
+    Writes the dataset through the user-defined processor based on the metadata of dataset. The processor writes dataset
+    to external system and generates the :class:`~ai_flow.meta.ai_graph.ai_node.WriteDatasetNode` with
```

Comment:
full class name should be fixed:
```
:class:`~ai_flow.meta.ai_graph.ai_node.WriteDatasetNode `" -> :class:`~ai_flow.ai_graph.ai_node.WriteDatasetNode
```

There are some wrong usage in the file as well. Maybe we can do a global replacement to fix the issue.